src/Nethermind/Nethermind.Merge.Plugin/Data/GetPayloadV2Result.cs

```diff
@@ -17,7 +16,7 @@ public GetPayloadV2Result(Block block, UInt256 blockFees)
 
     public UInt256 BlockValue { get; }
 
-    public ExecutionPayload ExecutionPayload { get; }
+    public virtual ExecutionPayload ExecutionPayload { get; }
```

Comment:
maybe generic instead of `virtual`? (Not convinced here)

Reply:
Was my first thought too, but we have `ExecutionPayload = new(block);` above which is not possible to do if ExecutionPayload is a generic type.